.changeset/empty-pillows-hunt.md

```diff
@@ -0,0 +1,152 @@
+---
+'@primer/react': major
+---
+
+### ActionList
```

Comment:
This is a duplicate of `.changeset/deprecate-actionlistv1-promote-actionlistv2.md`